lib/socket-pool.js

```diff
@@ -117,7 +117,7 @@ class SocketRef {
       .on('message', this._onmessage.bind(this))
       .on('idle', this._onidle.bind(this))
       .on('busy', this._onbusy.bind(this))
-      .bind()
+      .bind(0, pool._dht.io._host)
```

Comment:
this is accessing a private prop. feauture is okay, but then dht-rpc should expose something like dht.listening (null or the host its bound to) etc (naming tdb)

Reply:
Actually thinking this through, this doesn't make sense currently, as it needs much more information that just the "parent host". The swarm runs (not fully yet but will) in dual mode, meaning it uses ipv6 and ipv4 and dht-rpc only runs in the ipv4 so this turns massively complex really fast.

The solution WHEN we enable dual mode is that hyperswarm-dht has options for choosing what interfaces it prefers for both and passes them back up to dht-rpc, ie the other way